src/catplus-common/src/graph/namespaces/qudt.rs

```diff
@@ -9,4 +9,5 @@ namespace! {
 }
 lazy_static! {
     pub static ref ns: Namespace<&'static str> = Namespace::new(PREFIX.as_str()).unwrap();
+    pub static ref ns_vocab: Namespace<&'static str> = Namespace::new("http://qudt.org/vocab/unit/").unwrap();
```

Comment:
I don't think that line is needed: compare to the other namespaces: just the line above should be able to do the mapping.